jenkins/flow/pr/edx-platform-python-unittests-pr.groovy

```diff
@@ -8,31 +8,38 @@ def subsetJob = build.environment.get("SUBSET_JOB") ?: "edx-platform-test-subset
 def repoName = build.environment.get("REPO_NAME") ?: "edx-platform"
 def coverageJob = build.environment.get("COVERAGE_JOB") ?: "edx-platform-unit-coverage"
 def workerLabel = build.environment.get("WORKER_LABEL") ?: "jenkins-worker"
+def djangoVersion = build.envrironment.get("DJANGO_VERSION") ?: ""
+
+// Any environment variables that you want to inject into the environment of
+// child jobs of this build flow should be added here (comma-separated,
+// in the format VARIABLE=VALUE)
+// following v
+def envVarString = "DJANGO_VERSION=${djangoVersion}"
 
 guard{
     unit = parallel(
       {
-        lms_unit_1 = build(subsetJob, sha1: sha1, SHARD: "1", TEST_SUITE: "lms-unit", PARENT_BUILD: "PR Build #" + build.number, WORKER_LABEL: workerLabel)
+        lms_unit_1 = build(subsetJob, sha1: sha1, SHARD: "1", TEST_SUITE: "lms-unit", PARENT_BUILD: "PR Build #" + build.number, WORKER_LABEL: workerLabel, ENV_VARS: envVarString)
         toolbox.slurpArtifacts(lms_unit_1)
       },
       {
-        lms_unit_2 = build(subsetJob, sha1: sha1, SHARD: "2", TEST_SUITE: "lms-unit", PARENT_BUILD: "PR Build #" + build.number, WORKER_LABEL: workerLabel)
+        lms_unit_2 = build(subsetJob, sha1: sha1, SHARD: "2", TEST_SUITE: "lms-unit", PARENT_BUILD: "PR Build #" + build.number, WORKER_LABEL: workerLabel, ENV_VARS: envVarString)
         toolbox.slurpArtifacts(lms_unit_2)
       },
       {
-        lms_unit_3 = build(subsetJob, sha1: sha1, SHARD: "3", TEST_SUITE: "lms-unit", PARENT_BUILD: "PR Build #" + build.number, WORKER_LABEL: workerLabel)
+        lms_unit_3 = build(subsetJob, sha1: sha1, SHARD: "3", TEST_SUITE: "lms-unit", PARENT_BUILD: "PR Build #" + build.number, WORKER_LABEL: workerLabel, ENV_VARS: envVarString)
         toolbox.slurpArtifacts(lms_unit_3)
       },
       {
-        lms_unit_4 = build(subsetJob, sha1: sha1, SHARD: "4", TEST_SUITE: "lms-unit", PARENT_BUILD: "PR Build #" + build.number, WORKER_LABEL: workerLabel)
+        lms_unit_4 = build(subsetJob, sha1: sha1, SHARD: "4", TEST_SUITE: "lms-unit", PARENT_BUILD: "PR Build #" + build.number, WORKER_LABEL: workerLabel, ENV_VARS: envVarString)
         toolbox.slurpArtifacts(lms_unit_4)
       },
       {
-        cms_unit = build(subsetJob, sha1: sha1, SHARD: "1", TEST_SUITE: "cms-unit", PARENT_BUILD: "PR Build #" + build.number, WORKER_LABEL: workerLabel)
+        cms_unit = build(subsetJob, sha1: sha1, SHARD: "1", TEST_SUITE: "cms-unit", PARENT_BUILD: "PR Build #" + build.number, WORKER_LABEL: workerLabel, ENV_VARS: envVarString)
         toolbox.slurpArtifacts(cms_unit)
       },
       {
-        commonlib_unit = build(subsetJob, sha1: sha1, SHARD: "1", TEST_SUITE: "commonlib-unit", PARENT_BUILD: "PR Build #" + build.number, WORKER_LABEL: workerLabel)
+        commonlib_unit = build(subsetJob, sha1: sha1, SHARD: "1", TEST_SUITE: "commonlib-unit", PARENT_BUILD: "PR Build #" + build.number, WORKER_LABEL: workerLabel, ENV_VARS: envVarString)
```

Comment:
Nit: If there was any equivalent functionality to Python's `functools.partial` in Groovy, this code would be a good candidate for it!